cmd/osv-scanner/main.go

```diff
@@ -99,6 +103,11 @@ func run(args []string, stdout, stderr io.Writer) int {
 				Usage: "attempt call analysis on code to detect only active vulnerabilities",
 				Value: false,
 			},
+			&cli.StringFlag{
+				Name:      "experimental-diff",
```

Comment:
Fine as an experimental flag for now, but should we consider breaking this out into its own binary? It seems like something that would only ever be used in the CI/PR scanning case.

Reply:
That's a good point, happy to move it to another binary.